snuba/query/dsl_mapper.py

```diff
@@ -37,14 +40,26 @@ def and_cond_repr(exp: Expression, visitor: ExpressionVisitor[str]) -> str:
     return f"and_cond({parameters})"
 
 
+or_cond_match = FunctionCallMatch(
+    StringMatch("or"),
+)
+
+
+def or_cond_repr(exp: Expression, visitor: ExpressionVisitor[str]) -> str:
```

Comment:
why do you need this if we can just use the functions helper and generate basically the same thing?